controller/src/main/java/org/jboss/as/controller/operations/global/GlobalOperationAttributes.java

```diff
@@ -90,4 +93,11 @@
     .setAllowNull(true)
     .build();
 
+    static final AttributeDefinition GROUP_ELEMENT = new SimpleAttributeDefinitionBuilder(ModelDescriptionConstants.GROUP_ELEMENT, ModelType.STRING, true)
+    .setAllowNull(false)
+    .build();
+
+    static final SimpleListAttributeDefinition ATTRIBUTE_GROUP = new SimpleListAttributeDefinition.Builder(ModelDescriptionConstants.NAME, GROUP_ELEMENT)
+    .setAllowNull(true)
```

Comment:
This should have a ParameterCorrector associated such that it is tolerant of a value of ModelType.STRING instead of ModelType.LIST, with the correcting converting the string to a single element list. For example, see

https://github.com/wildfly/wildfly-core/blob/master/server/src/main/java/org/jboss/as/server/controller/resources/DeploymentAttributes.java#L166

and 

https://github.com/wildfly/wildfly-core/blob/master/server/src/main/java/org/jboss/as/server/controller/resources/DeploymentAttributes.java#L415